### ISSUE TITLE
Emailing with PSWriteHTML Conflicts with McAfee

### ISSUE THREAD

**Issue by mharding71** [EvotecIT/PSWriteHTML#174]
Hi,
With additional security in my domain, McAfee is flagging the Email function as a security risk. Have you seen this before? Is there a way to troubleshoot this to eliminate this issue? 

User ran C:\Windows\System32\WindowsPowerShell\v1.0\powershell_ise.exe, which attempted to access the process lsass.exe, violating the rule "Suspicious LSASS Access from Powershell", and was blocked. For information about how to respond to this event, see KB85494.

**Comment by PrzemyslawKlys** [comment]
I saw once a similar situation with windows defender -> https://evotec.xyz/import-module-this-script-contains-malicious-content-and-has-been-blocked-by-your-antivirus-software/

AMSI was triggering problems on install-module, which was gone as soon as definitions were updated. 

You may try that or report to McAfee as false positive. Don't know any way to "debug" on which command exactly it's acting up. It may be because of signature. Really hard to say.

**Comment by mharding71** [comment]
OMG, thanks for the quick response. Crazy that you're on it like that. I thought I was getting better at PowerShell until I ran into your scripts and coding. Hats off to the work you guys do.

So, I believe I know what the issue is/was, but won't know for sure unless I try to reproduce. So I was using the code initially and it was failing because I missed setting the port to 25 for SMTP with no login. The code ran successfully but I was not getting the email. I then installed Emailimo to attempt to see if that was the issue, but that is when I noticed the logs for McAfee flagging the code as suspicious. I then uninstalled Emailimo, used port 25, and used the debug "suppress" switch and it gave me a lot of information about the process. After many attempts, I got it working but my new issue is the format. I went back to your pages many times and realize I missed that one line that the formatting I come to expect from PSWriteHTML is only in the HTML attached. I don't see it in the email. So I wasted my time completely because I already have several reports sent as attached HTML and they're all great. 

With that said, thanks for your help. I'm responding to help out the next person. It's also quite possible that the McAfee application was flagging it as malicious and then decided to stop. Either way, it is now not flagging it and working fine.
